dashboards/traefik/traefik.json

```diff
@@ -165,7 +165,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "FROM Metric SELECT\nsum(traefik_service_requests_total) AS 'Total Requests'\nFACET instance, service\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write'"
+                "query": "FROM Metric SELECT\nsum(traefik_service_requests_total) AS 'Total Requests'\nFACET instance, service\nLIMIT MAX\nWHERE instrumentation.name = 'remote-write' AND prometheus_server IN ( {{prometheus_server}}) AND instance IN ( {{instance}} ) facet concat(prometheus_server,' - ',instance)"
```

Comment:
2 `FACET` conditions in this query; recommend consolidating into:

```
FACET concat(prometheus_server,' - ',instance) AS 'Instance', service
```

Reply:
Thanks, changes made.